src/main/java/pl/touk/sputnik/connector/gerrit/ReviewInputBuilder.java

```diff
@@ -29,21 +29,24 @@ public ReviewInput toReviewInput(@NotNull Review review, @Nullable String tag) {
         if (StringUtils.isNotBlank(tag)) {
             reviewInput.tag = tag;
         }
-        reviewInput.comments = review.getFiles().stream()
+        reviewInput.robotComments = review.getFiles().stream()
                 .collect(Collectors.toMap(ReviewFile::getReviewFilename, this::buildFileComments));
         return reviewInput;
     }
 
     @NotNull
-    private List<ReviewInput.CommentInput> buildFileComments(@NotNull ReviewFile reviewFile) {
+    private List<ReviewInput.RobotCommentInput> buildFileComments(@NotNull ReviewFile reviewFile) {
         return reviewFile.getComments().stream()
                 .map(this::buildCommentInput)
                 .collect(Collectors.toList());
     }
 
     @NotNull
-    private ReviewInput.CommentInput buildCommentInput(Comment comment) {
-        ReviewInput.CommentInput commentInput = new ReviewInput.CommentInput();
+    private ReviewInput.RobotCommentInput buildCommentInput(Comment comment) {
+        ReviewInput.RobotCommentInput commentInput = new ReviewInput.RobotCommentInput();
+
+        commentInput.robotId = "sputnik";
+        commentInput.robotRunId = UUID.randomUUID().toString();
```

Comment:
This will generate robotRunId unique for every comment. Maybe it should keep same runId for all comments of this "run"?